src/renderer/containers/ReplayBrowser/index.tsx

```diff
@@ -46,6 +46,7 @@ export const ReplayBrowser: React.FC = () => {
   const fileErrorCount = useReplays((store) => store.fileErrorCount);
   const rootSlpPath = useSettings((store) => store.settings.rootSlpPath);
   const { addToast } = useToasts();
+  const [list, setList] = React.useState<Array<string>>([]);
```

Comment:
Having the store here means that changing pages will clear the list, since the list gets reset on mount/unmount. You should instead modify `src/renderer/store/replays/index.ts` to add a list for selected items, and corresponding method for toggling the selection etc. That way the list state gets preserved even when changing pages.